zanshincli/main.py

```diff
@@ -1015,6 +1014,18 @@ def scan_target_groups_list(organization_id: UUID = typer.Argument(..., help="UU
     client = Client(profile=global_options['profile'])
     output_iterable(client.iter_organization_scan_target_groups(organization_id))
 
+@scan_target_group_app.command(name='create')
+def scan_target_groups_create(
+        organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+        kind: ScanTargetKind = typer.Argument(..., help="kind of the scan target group. Should be 'ORACLE'" ),
+        name: str = typer.Argument(..., help="name of the scan target group")
+):
+    """
+    Create a scan target group of the organization.
```

Comment:
s/of/for/

Reply:
sorry, I didn't understand

Reply:
Ah, using `sed` syntax: replace "of" for "for":

"Creates a scan target group for the organization"

Reply:
oh ok, it's done now